package.json

```diff
@@ -387,6 +387,11 @@
     "embedding-sdk:generate-changelog": "yarn conventional-changelog -p angular --pkg enterprise/frontend/src/embedding-sdk/package.template.json --config enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js --tagPrefix embedding-sdk-",
     "embedding-sdk:publish": "cd ./resources/embedding-sdk && npm publish",
     "embedding-sdk:test-unit": "yarn test-unit enterprise/frontend/src/embedding-sdk/",
+    "embedding-sdk:tsc": "tsc --project tsconfig.sdk.json",
```

Comment:
I think we should eventually move to our own package.json file to isolate our scripts... we have something like 17 scripts and they're not even in the same "cluster" in the package.json as they're sorted alphabetically and they don't all have the same prefix.
(Just cmd+f for "sdk" in the package.json and you'll see what I mean)